plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorSmokeTest.java

```diff
@@ -413,4 +432,8 @@ protected String getColumnComment(String tableName, String columnName)
     protected abstract void dropTableFromMetastore(String tableName);
 
     protected abstract String getMetadataLocation(String tableName);
+
+    protected abstract String schemaPath();
+
+    protected abstract boolean locationExists(String tableLocation);
```

Comment:
I would rename `tableLocation` to `location`. This method can be used for verifying location in general. No need to limit the usage to "table" location.